doc/tutorials/cpu_sharing.rst

```diff
@@ -21,13 +21,12 @@ The below diagram shows that the vCPU layer invokes APIs provided by scheduling
 .. figure:: images/cpu_sharing_api.png
    :align: center
 
-vCPU affinity
+CPU affinity
 *************
 
-Currently, we do not support vCPU migration; the assignment of vCPU mapping to pCPU is statically configured in the VM configuration via a vcpu_affinity array. The item number of the array matches the vCPU number of this VM. Each item has one bit to indicate the assigned pCPU of the corresponding vCPU. Use these rules to configure the vCPU affinity:
+Currently, we do not support vCPU migration; the assignment of vCPU mapping to pCPU is fixed at the time the VM is launched. The statically configured cpu_affinity_bitmap in the VM configuration defines a superset of pCPUs that the VM is allowed to run on. One bit in this bitmap indicates that one pCPU could be assigned to this VM, and the bit number is the pCPU ID. A pre-launched VM is supposed to be launched on exact number of pCPUs that are assigned in this bitmap. and the vCPU to pCPU mapping is implicitly indicated: vCPU0 maps to the pCPU with lowest pCPU ID, vCPU1 maps to the second lowest pCPU ID, and so on.
```

Comment:
please keep line length < 80 characters, also delete the leading space on this sentence.

Reply:
There are a number of paragraphs in this doc are written in this style. I can change all of them and update the PR.